packages/browser/scripts/generate.ts

```diff
@@ -1,13 +1,12 @@
-import spawn from 'nano-spawn';
-import fs from 'fs-extra';
 import { fileURLToPath } from 'node:url';
 import { dirname, join, resolve, sep } from 'node:path';
 import { sep as posixSep } from 'node:path/posix';
+import { readdir, mkdir } from 'node:fs/promises';
 
 // Fetch latest version
 
 console.log('Getting latest version of \x1b[36m@types/chrome\x1b[0m');
-await spawn('pnpm', ['i', '--ignore-scripts', '-D', '@types/chrome@latest']);
+await Bun.$`bun install --ignore-scripts -D @types/chrome@latest`;
```

Comment:
That change should fix https://github.com/nodejs/help/issues/5063 
Which we have on the latest version of node.